test/envoye2e/stackdriver_plugin/fake_stackdriver.go

```diff
@@ -143,11 +143,12 @@ func (s *LoggingServer) WriteLogEntries(ctx context.Context, req *logging.WriteL
 	s.mux.Lock()
 	defer s.mux.Unlock()
 	for _, entry := range req.Entries {
-		// Add the general labels to every log entry in list logentries response.
+		// Add the general labels and log name to every log entry in list logentries response.
 		tmpEntry := proto.Clone(entry).(*logging.LogEntry)
 		for k, v := range req.Labels {
 			tmpEntry.Labels[k] = v
 		}
+		tmpEntry.Labels["log_name"] = req.LogName
```

Comment:
We could also use `LogName` field in LogEntry(https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry) ?

Reply:
Thanks done!